src/request-pipeline/context.ts

```diff
@@ -296,6 +299,13 @@ export default class RequestPipelineContext {
             .map(userScript => userScript.url);
     }
 
+    private async _getDestResBody (res: IncomingMessageLike | http.IncomingMessage | FileStream): Promise<Buffer> {
+        if (IncomingMessageLike.isIncomingMessageLike(res))
+            return (res as IncomingMessageLike).getBody();
+
+        return await fetchBody(this.destRes);
```

Comment:
Minor: I think we can get rid of using `await` here. Reason: [Why Using `return await` Is a Bad Idea?](http://hassansin.github.io/Why-return-await-Is-a-Bad-Idea)

Reply:
Ok